content/en/integrations/guide/prometheus-metrics.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Mapping Prometheus Metrics to Datadog Metrics
+kind: guide
+aliases:
+  - /integrations/faq/how-to-collect-metrics-with-sql-stored-procedure/
+further_reading:
+- link: "https://www.datadoghq.com/blog/sql-server-metrics/#create-a-stored-procedure-to-generate-and-collect-metrics"
+  tag: "Blog"
+  text: "Create a stored procedure to generate and collect metrics"
+- link: "/integrations/mysql/"
+  tag: "Documentation"
+  text: "Datadog-MySQL integration"
+---
+
+If you are using Datadog's Prometheus or OpenMetrics checks, you may be interested in how these metrics map to existing Datadog metrics types. 
+
+## Prometheus and OpenMetrics metric types
+
+* `counter`: a cumulative metric that represents a single monotonically increasing counter, whose value can only increase—or be reset to zero.
+* `gauge`: a metric that represents a single numerican value, which can arbitrarily go up and down.
+* `histogram`: samples observations and counts them in configurable buckets; also provides a sum of all observed values.
+* `summary`: similar to histogram; samples observations, provides a sum of all observed values, and calculates configurable quantiles over a sliding time window.
+
+## How they map to Datadog metrics
```

Comment:
Can we rephrase the title or make it more clear in the following paragraph to which metric types we are referring to? 

It can be confusing because the first link below links to Datadog Metric Types, and then the following metric types refer to OpenMetrics/Prometheus Metric Types